backend/tilavarauspalvelu/models/reservation_series/actions.py

```diff
@@ -92,6 +92,8 @@ def pre_calculate_slots(
                 delta += 7
 
             begin_date: datetime.date = self.reservation_series.begin_date + datetime.timedelta(days=delta)
+            if begin_date > self.reservation_series.end_date:
```

Comment:
? or `< ...start_date`

Reply:
Too terse, can you explain?

Reply:
Was just note to self (before meeting) to check further on weekdays before the given range, but should be ok because of `delta += 7`